src/components/views/avatars/RoomAvatar.js

```diff
@@ -22,7 +23,8 @@ module.exports = React.createClass({
     displayName: 'RoomAvatar',
 
     propTypes: {
-        room: React.PropTypes.object.isRequired,
+        room: React.PropTypes.object,
```

Comment:
it would be nice to add some comments to explain how `room` and `oobData` interact. What if I set both? What if I set neither?